c2corg_ui/templates/helpers/view.html

```diff
@@ -180,7 +180,11 @@ <h3 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" dat
            <app-card app-card-doc="doc"></app-card>
         % if showDelete:
         <app-delete-association child-doctype="${type}" parent-id="${document['document_id']}" child-id="doc.document_id"
-                                ng-if="userCtrl.auth.isAuthenticated()"></app-delete-association>
+                                ng-if="userCtrl.auth.isAuthenticated()
+                                % if document['doctype'] == 'routes':
+                                    && ${associations}.length > 1
+                                % endif
```

Comment:
Then OK for me :)
Feel free to merge and mark the related issue as "ready to be tested".